python/cog/server/useragent.py

```diff
@@ -0,0 +1,17 @@
+def _get_version() -> str:
+    try:
+        try:
+            from importlib.metadata import version
+        except ImportError:
+            pass
+        else:
+            return version("cog")
+        import pkg_resources
+
+        return pkg_resources.get_distribution("cog").version
+    except Exception:
+        return "unknown"
+
+
+def get_user_agent() -> str:
+    return f"cog-worker/{_get_version()}"
```

Comment:
why pull this out? in the async branch I've been doing more inlining and less indirection where this makes sense. is get_user_agent/_get_version used in more than one place?

Reply:
Hmm, in this branch it's not. But these files exist on main, so I tried to keep it consistent. I'm happy to inline it.